CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.46.0] - 2022-03-29
+### Changed
+- Treat HTTP 204 as ok status
```

Comment:
which services/endpoints return 204? That'd be more changelog-worthy :slightly_smiling_face: could be a bugfix or preparation for new endpoints, don't have context

Reply:
raster support in geospatial api (experimental) is returning 204 when the corresponding raster is empty (NULL in db). We considered using 404 (or 400) too for the same request but it felt more logical to go with 204 (No content)

Reply:
then suggest putting something like "fixed retrieval of empty raster in experimental geospatial api"

Reply:
https://pr-1632.specs.preview.cogniteapp.com/v1.json.html#operation/getRaster
We are returning 204 if there is no data for this endpoint.

This PR is more aligned with https://github.com/cognitedata/service-contracts/blob/master/design_guide/API_design_guide.md#status-codes where 204 should be supported.